src/person/types/params.ts

```diff
@@ -0,0 +1,29 @@
+import { Sort } from '../../common';
+import { IPerson, PersonTypeEnum } from './types';
+
+export interface PersonParams {
```

Comment:
```suggestion
export interface IPersonFilters {
```

On garde les préfix sur le nommage des interfaces sur ce projet, pour le moment.